packages/frontend/app/components/app/sidebar/actions/response-panel/index.ts

```diff
@@ -0,0 +1,26 @@
+import Component from '@glimmer/component';
+
+import { action } from '@ember/object';
+
+import Channel from 'emberclear/models/channel';
+
+interface IArgs {
+  channel: Channel;
+}
+
+export default class ResponsePanel extends Component<IArgs> {
+  @action
+  yes() {
+    alert('voting yes!');
```

Comment:
I'm assuming that these will eventually call the services to send votes. Because a response panel will in theory be tied to a single vote, would it be possible to pass that vote as a parameter so it is easier to manage? Also maybe add TODOs for where the code for sending votes will be needed.